domains/aman-sapkota.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "dev-amansapkota",
+    "email": "amansapkota14@gmail.com"
+  },
+  "records": [
+    {
+      "type": "CNAME",
+      "value": "aman-sapkota.github.io"
+    }
+  ]
```

Comment:
Improperly done CNAME record, this is not how it is processed. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.